WasatchNET/HOCTSpectrometer.cs

```diff
@@ -315,6 +315,10 @@ public static ushort[] CaptureBitMap(int iFramesTransmitted, bool bInternalTrigg
                     }
                     else
                     {
+                        if (GetFifoStatus())
+                        {
+                            Console.WriteLine("buh buh buh {0}", iFifoProcessingCount);
```

Comment:
Remember that this is customer sample code.

Reply:
haha, very good point